README.md

```diff
@@ -32,10 +32,13 @@ We welcome [contributions](#contributing) to this repository. Within it you will
     - the [explanatory documentation](https://blockprotocol.org/docs) at [src/\_pages/docs](https://github.com/blockprotocol/blockprotocol/tree/main/site/src/_pages/docs)
 
 1.  `packages`: utility packages for constructing Block Protocol blocks. These are all available via `yarn add <package_name>` or `npm install <package_name>`. See the individual README files in each folder for more details.
-    - `block-template`: a template for a React-based block
-    - `blockprotocol`: TypeScript types for Block Protocol properties and functions, as described in the spec
+    - `@blockprotocol/core`: implements the Block Protocol core specification for blocks and embedding applications
+    - `@blockprotocol/graph`: implements the Block Protocol graph specification for blocks and embedding applications
+    - `block-scripts`: scripts used by `block-template`
+    - `block-template`: starter templates for blocks, used by `create-block-app`
```

Comment:
Not sure about this move – I haven’t seen multiple templates co-located in a single package. I’d go with one template per package or even keep templates in the repo instead of npm (like [Next.js examples](https://github.com/vercel/next.js/tree/canary/examples)).

Co-locating templates in single package means that users will see version bump for React template while we’ve only updated web components code. That’s confusing.

Some thoughts on this: [Slack](https://hashintel.slack.com/archives/C02LG39FJAU/p1652379269733239?thread_ts=1652165331.405139&cid=C02LG39FJAU) (internal)

Reply:
I took this from `create-next-app`, which has multiple templates co-located in a single package, albeit just JS and TS.

Do we think it's likely that users are going to get confused by this? They aren't interacting with this package directly, after all, but via `create-block-app`. And if we have templates separately in the repo, there will be no version bumps at all.

If we think this is going to be an actual problem, can we split them out in a future PR?